### PR TITLE
More letters and colors

### PR DESCRIPTION
**Pull request by awWhy**
Nothing Special

### CODE REVIEW

File: src/Cubes.ts [hunk 1]
```diff
@@ -161,14 +161,14 @@ const cubeUpgradeDescriptions = [
     '[5x1] Yeah yeah yeah, +5% score on Ascension per level. Isn\'t it enough?',
     '[5x2] You now gain +4% Obtainium per level, which is not dependent on corruptions!',
     '[5x3] Gain another +3% corruption-independent Obtainium per level.',
-    '[5x4] Blah blah blah Midas works harder (same rewards as before)',
-    '[5x5] Blah blah blah Hermes works harder (same rewards as before)',
+    '[5x4] Blah blah blah Midas works harder (Same rewards as before)',
```
Comment (KhafraDev) [inline src/Cubes.ts:164]: s

File: src/Cubes.ts [hunk 2]
```diff
@@ -161,14 +161,14 @@ const cubeUpgradeDescriptions = [
...
+    '[5x5] Blah blah blah Hermes works harder (Same rewards as before)',
```
Comment (KhafraDev) [inline src/Cubes.ts:165]: s

File: src/Cubes.ts [hunk 3]
```diff
@@ -161,14 +161,14 @@ const cubeUpgradeDescriptions = [
...
     '[5x6] Gain +5% more Offerings per level!',
     '[5x7] Gain +10% more Obtainium per level!',
-    '[5x8] When you ascend, start with 1 worker Ant (this is a lot better than it sounds!)',
-    '[5x9] When you ascend, gain 1 of each Challenge 6-8 completion.',
+    '[5x8] When you Ascend, start with 1 Worker Ant (This is a lot better than it sounds!)',
```
Comment (KhafraDev) [inline src/Cubes.ts:168]: t

File: src/Toggles.ts [hunk 4]
```diff
@@ -729,10 +729,10 @@ export const toggleMaxBuyCube = () => {
     const el = DOMCacheGetOrSet('toggleCubeBuy')
     if (G['buyMaxCubeUpgrades']) {
         G['buyMaxCubeUpgrades'] = false;
-        el.textContent = 'Upgrade: 1 Level wow'
+        el.textContent = 'Upgrade: 1 Level Wow!'
```
Comment (KhafraDev) [inline src/Toggles.ts:732]: will this be confused with wow cubes?

File: src/Toggles.ts [hunk 5]
```diff
@@ -729,10 +729,10 @@ export const toggleMaxBuyCube = () => {
...
     } else {
         G['buyMaxCubeUpgrades'] = true;
-        el.textContent = 'Upgrade: MAX [if possible wow]'
+        el.textContent = 'Upgrade: MAX [If possible Wow!]'
```
Comment (KhafraDev) [inline src/Toggles.ts:735]: ditto & i

File: src/Toggles.ts [hunk 6]
```diff
@@ -903,9 +903,9 @@ export const toggleAscStatPerSecond = (id: number) => {
 
 export const toggleHepteractAutoPercentage = async(): Promise<void> => {
     const amount = await Prompt(
-        'Enter a number from 0 to 100 (integer only!) to set autocraft percentage. ' +
-        'Every ascension, that percentage of your hepteracts are used to craft equally split ' +
-        'between every hepteract with AUTO ON. Auto crafting also does not consume other resources! ' +
+        'Enter a number from 0 to 100 (Integer only!) to set autocraft percentage. ' +
```
Comment (KhafraDev) [inline src/Toggles.ts:906]: i

**Review by KhafraDev — CHANGES_REQUESTED** [review]

**Comment by awWhy** [comment]
I dont know what changes you mean. All other () cases start with Big letter
To have button say Wow! in Wow! cube tab fits theme more

File: src/Cubes.ts (hunk 2 above)
Comment (awWhy) [reply src/Cubes.ts:165]: Not changin, as every other simular case have Big letter inside () or []

File: src/Toggles.ts (hunk 4 above)
Comment (awWhy) [reply src/Toggles.ts:732]: Its a Wow! cube tab. Fits theme more to have Wow!
What will people think, that they will upgrade cubes themselfs if they press button?

File: src/Cubes.ts (hunk 2 above)
Comment (KhafraDev) [reply src/Cubes.ts:165]: then it's not getting merged

**Comment by awWhy** [comment]
I'm just going to close it and remove parts Khafra dont like

**Pull request closed without merging**